<launch>
    <arg
            name="camera_info_url"
            default="file://$(find hardware_interfaces)/config/calib_ov9281_00.yaml" />
    <arg
            name="camera_name"
            default="ov9281_00" />
    <arg
            name="camera_frame_id"
            default="camera_frame" />
    <arg
            name="framerate"
            default="10" />

    <node
            pkg="hardware_interfaces"
            type="camera_node"
            name="$(arg camera_name)">
        <param
                name="camera_name"
                value="$(arg camera_name)" />
        <param
                name="camera_frame_id"
                value="$(arg camera_frame_id)" />
        <param
                name="camera_info_url"
                value="$(arg camera_info_url)" />
    </node>
</launch>
